packages/react/src/components/DashboardEditor/DashboardEditor.test.e2e.jsx

```diff
@@ -184,7 +184,7 @@ describe('DashboardEditor', () => {
         expect(mockOnCardChange).to.be.calledWith(
           {
             ...card,
-            size: CARD_SIZES.MEDIUMWIDE,
+            size: CARD_SIZES.MEDIUMTHICK,
```

Comment:
What is the reason we are changing this test?

Reply:
The is a resize test. Previously, after resize `MEDIUM` should resize to `MEDIUMWIDE` which is the next one. But now we introduce `MEDIUMTHICK` between `MEDIUM` and `MEDIUMWIDE`, resize goes to `MEDIUMTHICK` as the next one. 

Cypress failed with errors like this:
![image](https://user-images.githubusercontent.com/24279794/179423346-3ae3bfcb-abc4-4ee7-825d-558dfe654f11.png)